src/main/java/nl/recognize/dwh/application/loader/EntityLoader.java

```diff
@@ -56,4 +56,9 @@ public interface EntityLoader {
      * - If you want to serialize an array of primitive types, you can use array_type
      */
     EntityMapping getEntityMapping();
+
+    /**
+     * Specify the data type of the identifier.
+     */
+    String getIdentifierType();
```

Comment:
Kan dat niet een enum zijn eigenlijk? Misschien iets voor later?

Reply:
Ja, dit moet eigenlijk een enum zijn, maar heb hem in lijn gehouden met het data type dat we overal gebruiken.